sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/AccessControlChanges.java

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.datalake.models;
+
+/**
+ * AccessControlChanges contains batch and cumulative counts of operations that change Access Control Lists recursively.
+ * Additionally it exposes path entries that failed to update while these operations progress.
+ */
+public class AccessControlChanges {
+    private Iterable<AccessControlChangeFailure> batchFailures;
+    private AccessControlChangeCounters batchCounters;
+    private AccessControlChangeCounters aggregateCounters;
+    private String continuationToken;
+
+    /**
+     * Returns an iterable of path entries that failed to update Access Control List within a single batch.
+     *
+     * @return An iterable of path entries that failed to update Access Control List within a single batch.
+     */
+    public Iterable<AccessControlChangeFailure> getBatchFailures() {
+        return batchFailures;
+    }
+
+    /**
+     * Sets an iterable of path entries that failed to update Access Control List within a single batch.
+     *
+     * @param batchFailures An iterable of path entries that failed to update Access Control List within a single batch.
+     * @return The updated object
+     */
+    public AccessControlChanges setBatchFailures(Iterable<AccessControlChangeFailure> batchFailures) {
+        this.batchFailures = batchFailures;
+        return this;
+    }
+
+    /**
+     * Returns an {@link AccessControlChangeCounters} that contains counts of paths changed within a single batch.
+     *
+     * @return {@link AccessControlChangeCounters}
+     */
+    public AccessControlChangeCounters getBatchCounters() {
+        return batchCounters;
+    }
+
+    /**
+     * Sets an {@link AccessControlChangeCounters} that contains counts of paths changed within a single batch.
+     *
+     * @param batchCounters {@link AccessControlChangeCounters}
+     * @return The updated object.
+     */
+    public AccessControlChanges setBatchCounters(AccessControlChangeCounters batchCounters) {
+        this.batchCounters = batchCounters;
+        return this;
+    }
+
+    /**
+     * Returns an {@link AccessControlChangeCounters} that contains counts of paths changed from start of the operation.
+     *
+     * @return {@link AccessControlChangeCounters}
+     */
+    public AccessControlChangeCounters getAggregateCounters() {
+        return aggregateCounters;
+    }
+
+    /**
+     * Sets an {@link AccessControlChangeCounters} that contains counts of paths changed from start of the operation.
+     *
+     * @param aggregateCounters {@link AccessControlChangeCounters}
+     * @return The updated object.
+     */
+    public AccessControlChanges setAggregateCounters(AccessControlChangeCounters aggregateCounters) {
+        this.aggregateCounters = aggregateCounters;
+        return this;
+    }
+
+    /**
+     * Returns the continuation token.
+     * <p>
```

Comment:
This needs a </p> to end the paragraph somewhere. Same for the rest of the PR where there's lonely <p>'s

Reply:
This is standard for how we write javadocs